lib/stormpath-sdk/resource/collection.rb

```diff
@@ -51,6 +51,10 @@ def limit limit
     self
   end
 
+  def size
+    current_page.items.size
```

Comment:
@ecrisostomo  size is removed and branch is updated. regarding the PaginatedIterator, not a problem will leave it as it is. It was just a suggestion.